lib/jormungandr/test/integration/Cardano/Wallet/Jormungandr/NetworkSpec.hs

```diff
@@ -261,7 +262,27 @@ spec = do
     describe "White-box error path tests" $
         around startNode $ do
 
-        it "can't get account state for account that does not exist" $
+        it "getAccountState: Invalid url path" $
+            \(_, url) -> do
+                manager <- newManager defaultManagerSettings
+                let wrongUrl = url { baseUrlPath = "/not-valid-prefix" }
+                let client = Jormungandr.mkJormungandrClient manager wrongUrl
+                res <- runExceptT $
+                    Jormungandr.getAccountState client testAccountId
+                res `shouldBe` Left
+                    (ErrGetAccountStateAccountNotFound testAccountId)
```

Comment:
I am wondering if there should not be some different error in such a case...

Reply:
Hmmm...  This is quite unlikely to happen in practice because the network layer is really only instantiated once we have fetched the initial genesis block from Jörmungandr. If we had a incorrect prefix at this stage, the server won't start and fail with a rather descriptive error. So I wouldn't worry too much about this particular case.